src/ResourceManager/Cdn/Commands.Cdn.Test/ScenarioTests/CustomDomainTests.ps1

```diff
@@ -61,9 +63,10 @@ Gets and removes custom domain with stopped endpoint
 #>
 function Test-CustomDomainGetRemoveWithStoppedEndpoint
 {
-    $endpointName = "sdktest-cee91bb1-996a-44f1-96e6-dceaa6707def"
-    $hostName = "sdktest-3acbafc1-3f50-4fa4-9132-6d8c944d67e9.azureedge-test.net"    
-    $customDomainName = getAssetName
+    $endpointName = "sdktest-b0939e74-75ba-4558-afe6-edc5c19ea713" 
```

Comment:
I get what you're saying, and believe me, I wanted to do this differently too, however the two CustomDomain tests rely on a non-injectable outside resource - a DNS CNAME that must be setup manually on DNS.  That's why the original authors ( @kuangweizhang ?  Qinyuan? ) did it this way.